source/agora/node/API.d

```diff
@@ -115,7 +115,7 @@ public interface API
             Return true if the node has this transaction hash.
 
         API:
-            GET /hasTransactionHash
+            GET /has_transaction_hash?tx=
```

Comment:
Other places don't include the query parameters. If we want to add them, it should be done for all of them (in a separate commit).

Reply:
I Fixed /has_transaction_hash